PaintingTrivia.py

```diff
@@ -119,14 +128,49 @@ def show_entries():
             'selected_painter_id'] else 'wrong_guesses'
         session[key] += 1
 
-    painters = Painter.query.order_by(func.random()).limit(4).all()
+    while True:
+        painters = Painter.query.order_by(func.random()).limit(4).all()
+        selected_painter = random.choice(painters)
+        selected_painting = Painting.query.filter(Painting.painter == selected_painter).order_by(func.random()).limit(
+            1).first()
+        if selected_painting:
+            break
```

Comment:
then a better solution would be to modify the query to only return the painters that have at least one picture.. we can do it tonight...